src/assets/css/main.css

```diff
@@ -386,3 +386,8 @@ br.big {
   -webkit-line-clamp: 3;
   -webkit-box-orient: vertical;
 }
+
+#tagline{
+
+  padding: 0px 0px 15px 0px;
```

Comment:
This doesn't seem necessary, why not use bootstraps `padding-` classes? The less custom css the better.

Reply:
good idea, wasn't aware of available styles

Reply:
No probs.